src/sdk/namespace_fs.js

```diff
@@ -1302,6 +1362,70 @@ class NamespaceFS {
         }
     }
 
+    /**
+     * Returns true if the entry path is symbolic link
+     * @param {*} fs_account_config 
+     * @param {*} entry_path 
+     * @returns 
+     */
+    async _is_symbolic_link_by_path(fs_account_config, entry_path) {
```

Comment:
I don't see this function being used anywhere. am I missing it?

Reply:
Removed it